README.md

```diff
@@ -80,7 +80,9 @@ Terragrunt users should also read [this page](https://www.infracost.io/docs/iac_
 
 #### `INFRACOST_TERRAFORM_BINARY`
 
-**Optional** Used to change the path to the `terraform` binary or version, should be set to the path of the Terraform or Terragrunt binary being used in Atlantis. If you're using the `infracost/infracost-atlantis` image (which is based on the [`runatlantis/atlantis`](https://github.com/runatlantis/atlantis/blob/master/Dockerfile) image), you can set this to any of the supported paths, e.g. `/usr/local/bin/terraform0.12.30` or `/usr/local/bin/terraform0.13.6`.
+**Required** Used to change the path to the `terraform` binary with the current version, should be set to the path of the Terraform or Terragrunt binary being used in Atlantis. If you're using the `infracost/infracost-atlantis` image (which is based on the [`runatlantis/atlantis`](https://github.com/runatlantis/atlantis/blob/master/Dockerfile) image), you can set this to any of the supported paths, e.g.
+`/usr/local/bin/terraform0.12.30` or
+`<data-dir>/bin/terraform<version>`. This should be from the path which atlantis keeps the terraform binary in.
```

Comment:
> This should be from the path which atlantis keeps the terraform binary in.

do you mean it's a relative path? what did you end-up setting the `INFRACOST_TERRAFORM_BINARY` to for your use-case?

Reply:
Yes, Actually it was not working with me except i set this value to the variable `<data-dir>/bin/terraform<version>`

Reply:
Thanks! I tweaked the wording a bit, do you think it's clear now? 
also, if you get the chance, plz take a quick look at https://github.com/infracost/infracost-atlantis/pull/12#issuecomment-832841678

Reply:
Yes , looks so good now.

Reply:
Awesome, I merged the PR. For future contributions, it would be good if you can take a minute to read/sign our Contributor License Agreement (https://cla-assistant.io/infracost/infracost-atlantis?pullRequest=12)